d2l-table-style.html

```diff
@@ -236,6 +236,151 @@
 			[dir="rtl"] d2l-table[no-column-border] d2l-tbody > d2l-tr > .d2l-table-cell-first {
 				border-right: var(--d2l-table-border);
 			}
+
+			[sticky-headers] {
+				position: relative;
+				overflow: hidden;
+			}
+
+			[sticky-headers] tr[header] th {
+				position: -webkit-sticky;
+				position: sticky;
+				top: 0;
+				border-bottom: var(--d2l-table-border);
+			}
+
+			[sticky-headers] th[sticky]:first-child,
+			[sticky-headers] td[sticky]:first-child {
```

Comment:
I think you'll want to leverage the existing CSS classes that we apply to the first/last rows and cells. i.e. `d2l-table-cell-first`, `d2l-table-cell-last`, `d2l-table-row-first` and `d2l-table-row-last`.

The reason is we often use row headers as the first cell in a row:
```
<table>
    <thead>
        <tr>
            <th>Col 1</th>
            <th>Col 2</td>
        </tr>
    </thead>
    <tbody>
    <tr>
        <th>Row header 1</th>
        <td>Row cell 1</td>
    </tr>
    <tr>
        <th>Row header 2</th>
        <td>Row cell 2</td>
    </tr>
    </tbody>
</table>
```

So your selectors for `:first-child` are going to trigger multiple times for those rows.